### PR TITLE
Auto testing

### PR DESCRIPTION
**Pull request by apackin**
#### What's this PR do?
Standardizes Elmer to V 1.0.0
Adds Istanbul test coverage and foundations for Travis/Coverall
#### How should this be manually tested?
npm run-script test-travis
#### Any background context you want to provide?
The .travis file is not being used yet.
Adding Travis to a repo requires admin permission for the repo. 
#### What are the relevant tickets?
https://admin.crisistextline.org/jira/secure/RapidBoard.jspa?rapidView=5&view=detail&selectedIssue=INT-264
https://admin.crisistextline.org/jira/secure/RapidBoard.jspa?rapidView=5&view=detail&selectedIssue=INT-219

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "application-name",
-  "version": "0.0.1",
+  "version": "1.0.0",
```
Comment (jrauschenberg) [inline package.json]: Okay, this works fine, we just have to make sure we update this everywhere.

File: package.json [hunk 2]
```diff
@@ -6,7 +6,8 @@
     "start": "./node_modules/.bin/forever start launch.json",
     "stop": "./node_modules/.bin/forever stop express; ./node_modules/.bin/forever stop cron; exit 0;",
     "restart": "./node_modules/.bin/forever restart express; ./node_modules/.bin/forever restart cron; exit 0;",
-    "test": "NODE_ENV=test ./node_modules/.bin/mocha --reporter spec 'test/index.js'"
+    "test": "NODE_ENV=test ./node_modules/.bin/mocha --reporter spec 'test/index.js'",
```
Comment (jrauschenberg) [inline package.json]: Okay, again nothing too new, I guess we're just waiting for the next step?

**Comment by apackin** [comment]
Green Light for testing.
Had to disable the Canvas based tests until we create the mock Canvas because we're not pushing our Canvas keys for Travis to use.

**Comment by tongxiang** [comment]
👍

**Pull request merged**